include/application/tasks/valve_demo.h

```diff
@@ -0,0 +1,19 @@
+/**
+ * This is a toy task for actuating a valve and demonstrating
+ * the platform's valve management functionality. This task
+ * does the following:
+ * 
+ *  1. Opens the inspiratory valve.

```

Comment:
isnt the valve already open. 

If so this demo will no permit to verify that the code works as expected and that the code to open a valve works